src/components/CTA/CTA.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FunctionComponent } from 'react';
+
+import classnames from 'classnames';
+
+import { DefaultProps } from '../../types';
+import { HeadingType } from '../../types/index';
+import { Button } from '../Button/Button';
+import { Heading } from '../Heading/Heading';
+import { IconName } from '../Icon/types';
+import { RichText } from '../RichText/RichText';
+
+import './CTA.scss';
+
+export interface CTAProps extends DefaultProps {
+	heading: string;
+	headingType: HeadingType;
+	content: string | string[];
+	buttonLabel: string;
+	buttonIcon?: IconName;
+}
+
+// TODO: Add button action
+export const CTA: FunctionComponent<CTAProps> = ({
+	className,
+	headingType = 'h3',
+	heading,
+	content,
+	buttonLabel,
```

Comment:
moest deze block slechts 1 knop hebben? of voorzien we voor alle blocks die een button hebben automatisch een button array editor, zodat ze altijd meerdere buttons kunnen toevoegen

Reply:
Voorlopig heb ik dit op Button staan omdat mijn Buttons Block nog niet af was, ik hang er een todo bij.